Sources/MapboxCoreNavigation/EHorizon/RoadGraph.swift

```diff
@@ -32,6 +32,35 @@ public final class RoadGraph {
         }
         return LineString(locations.map { $0.coordinate })
     }
+    
+    /**
+     Returns a line string geometry corresponding to the given path.
+     
+     - returns: A line string corresponding to the given path, or `nil` if any of path edges is inaccessible.
+     */
+    public func shape(of path: Path) -> LineString? {
+        let nativePath = GraphPath(edges: path.edgeIdentifiers.map { NSNumber(value: $0) },
+                                   percentAlongBegin: path.fractionFromStart,
+                                   percentAlongEnd: path.fractionToEnd,
+                                   length: path.length)
```

Comment:
Looks like this is a good candidate for convenience init method:
```swift
extension GraphPath {
    convenience init(_ path: RoadGraph.Path) {
        self.init(edges: path.edgeIdentifiers.map { NSNumber(value: $0) },
                  percentAlongBegin: path.fractionFromStart,
                  percentAlongEnd: path.fractionToEnd,
                  length: path.length)
    }
}

```

Reply:
For consistency, also a convenience initializer for GraphPosition based on Position.

Reply:
Thanks for the nit! I updated the code.

Reply:
Did you miss the push? I don't see the updated code.